viewing_party/party.py

```diff
@@ -0,0 +1,207 @@
+import math
+
+# wave 1 Creating new movie dictionary 
+
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if not title or not genre or not rating:
+        return None
+    else:
+        new_movie["title"] = title
+        new_movie["genre"] = genre 
+        new_movie["rating"] = rating
+    return new_movie
+
+# wave 1/2 Adding movie to the user's watched 
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+# wave 1/3 Adding movie to the user's watchlist 
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data 
+
+# wave 1/4 Remove title from a watchlist and add to watched 
+
+def watch_movie(user_data, title):
+    for index in range(len(user_data["watchlist"])):
+        if title == user_data["watchlist"][index]["title"]:
+            movie =(user_data["watchlist"][index])
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
```

Comment:
This is a little cleaner if you use a regular (non-`range`) `for` loop:

```suggestion
    for movie in user_data["watchlist"]:
        if title == movie["title"]:
            user_data["watchlist"].remove(movie)
            user_data["watched"].append(movie)
```